tools/scripts/linux.mk

```diff
@@ -11,3 +11,6 @@ CFLAGS +=  -g3 \
 $(PROJECT_TARGET):
 	$(MUTE) $(call mk_dir, $(BUILD_DIR)) $(HIDE)
 	$(MUTE) $(call set_one_time_rule,$@)
+
+linux_run: $(BINARY)
```

Comment:
Why not just run?

Reply:
when running `make run` this rule will be called

Reply:
not related to this commit but yeah I'm also wondering why this was implemented with [platform]_run: in generic,mk when you could simply have a run: rule in the [platform].mk

Reply:
I though was nice to search for the rules in one file (generic.mk)